doc/api/vm.md

```diff
@@ -304,29 +304,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
```

Comment:
@addaleax since this will be a docs-only deprecation for now, should I still update the YAML metadata?